### DISCUSSION TITLE
Not being able to retrieve IDPS used for encrypting in ps3

### DISCUSSION THREAD

**Discussion by TahaniSury** [LittleBigRefresh/Refresh#552] [Q&A/Support]
Im not entirely sure what this error means nor how to fix, this problem also follows up with the EBOOT failing to decrypt. Can someone help?

**Comment by jvyden** [comment]
You likely don't have WebMAN (full) installed on your PS3. Try installing the latest release on your PS3 from here: https://github.com/aldostools/webMAN-MOD/releases
